lib/apis/githubapi-types.d.ts

```diff
@@ -40,6 +40,31 @@ export interface StatusEvent {
 	sender: User;
 }
 
+export interface PushEventCommit {
+	sha: string;
+	message: string;
+	added: string[];
+	removed: string[];
+	modified: string[];
+	timestamp: string;
+}
+
+export interface PushEvent {
+	type: 'push';
+	action: string;
+	before: string;
```

Comment:
I've put links to the Github API docs in the relevant sections of this file. I don't really want to go and re-document what's documented by GH.